coremltools/test/sklearn_tests/test_composite_pipelines.py

```diff
@@ -49,14 +49,14 @@ def test_boston_OHE_plus_normalizer(self):
             result = evaluate_transformer(spec, input_data, output_data)
             assert result["num_errors"] == 0
 
-    def test_boston_OHE_plus_trees(self):
+    def _test_boston_OHE_plus_trees(self, loss='ls'):
```

Comment:
Would `pytest` still pick up this test case after changing the name?

Reply:
`pytest` will not pick up this test after the name change, but that is intentional.

I've basically just turned this function into a helper method that I'm calling from two other methods: `test_boston_OHE_plus_trees` tests the current functionality (i.e. what's currently in master) . 
`test_boston_OHE_plus_trees_with_huber_loss` tests the new functionality that I'm adding.